sdk/eventhub/eventhubs-checkpointstore-tables/src/tableCheckpointStore.ts

```diff
@@ -0,0 +1,92 @@
+   *
+   * @param fullyQualifiedNamespace - The fully qualified Event Hubs namespace. This is likely to be similar to
+   * <yournamespace>.servicebus.windows.net.
+   * @param eventHubName - The event hub name.
+   * @param consumerGroup - The consumer group name.
+   * @param options - A set of options that can be specified to influence the behavior of this method.
+   *  - `abortSignal`: A signal used to request operation cancellation.
+   *  - `tracingOptions`: Options for configuring tracing.
+   * @returns Partition ownership details of all the partitions that have had an owner.
+   */
+
+  async listOwnership(): /*
+        fullyQualifiedNamespace: string,
+        eventHubName: string,
+        consumerGroup: string,
+        options: OperationOptions = {}
+        */
+  Promise<PartitionOwnership[]> {
+    throw new Error("not implemented");
+  }
+
+  /**
+   * Claim ownership of a list of partitions. This will return the list of partitions that were
+   * successfully claimed.
+   *
+   * @param partitionOwnership - The list of partition ownership this instance is claiming to own.
+   * @param options - A set of options that can be specified to influence the behavior of this method.
+   *  - `abortSignal`: A signal used to request operation cancellation.
+   *  - `tracingOptions`: Options for configuring tracing.
+   * @returns A list partitions this instance successfully claimed ownership.
+   */
+  async claimOwnership(): /*
+    partitionOwnership: PartitionOwnership[],
+    options: OperationOptions = {}
+    */
+  Promise<PartitionOwnership[]> {
+    console.log("nothing to claim");
+    throw new Error("not implemented");
+  }
+
+  /**
+   * Lists all the checkpoints in a data store for a given namespace, eventhub and consumer group.
+   *
+   * @param fullyQualifiedNamespace - The fully qualified Event Hubs namespace. This is likely to be similar to
+   * <yournamespace>.servicebus.windows.net.
+   * @param eventHubName - The event hub name.
+   * @param consumerGroup - The consumer group name.
+   * @param options - A set of options that can be specified to influence the behavior of this method.
+   *  - `abortSignal`: A signal used to request operation cancellation.
+   *  - `tracingOptions`: Options for configuring tracing.
+   */
+  async listCheckpoints(): /*
+    fullyQualifiedNamespace: string,
+    eventHubName: string,
+    consumerGroup: string,
+    options: OperationOptions = {}
+    */
+  Promise<Checkpoint[]> {
+    console.log("no checkpoints to list");
+    throw new Error("not implemented");
+  }
+
+  /**
+   * Updates the checkpoint in the data store for a partition.
+   *
+   * @param checkpoint - The checkpoint.
+   * @param options - A set of options that can be specified to influence the behavior of this method.
+   *  - `abortSignal`: A signal used to request operation cancellation.
+   *  - `tracingOptions`: Options for configuring tracing.
+   * @returns The new etag on successful update.
```

Comment:
This comment isn't accurate, here's a suggestion:
```suggestion
   * @returns A promise that resolves when the checkpoint has been updated.
```